### PR TITLE
Welcome view

### PR DESCRIPTION
**Pull request by madhalle**
Welcome view front end version 1! Has welcome page graphics as well as nav bar.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -29,7 +29,8 @@ gem 'jbuilder', '~> 2.5'
 
 # Use ActiveStorage variant
 # gem 'mini_magick', '~> 4.8'
-
+gem 'bootstrap', '~> 4.5.0'
```
Comment (madhalle) [inline Gemfile:32]: Thought I was going to use this for nav bar, didn't end up using it but might come in handy later on

File: app/assets/stylesheets/application.css [hunk 2]
```diff
@@ -13,3 +13,70 @@
...
+  position: absolute;
+  margin-left: auto;
+  margin-right: auto;
+  padding-bottom: 200px;
+  opacity: .2 ;
+}
+
+
+
+@keyframes MoveUpDown {
```
Comment (madhalle) [inline app/assets/stylesheets/application.css:48]: allows for animation on page

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::Base
+
+  # def current_user?
```
Comment (madhalle) [inline app/controllers/application_controller.rb:3]: Will need something along the lines of this for nav bar to switch log in to log out and displau user name.

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -9,7 +9,54 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
+
+  <style>
```
Comment (madhalle) [inline app/views/layouts/application.html.erb:13]: Wondering if this styling can happen in application.css

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -9,7 +9,54 @@
...
+  }
+  </style>
+
+<body>
+    <nav>
+      <div class='navbar'>
+        <ul>
+          <li style="float:left"><a class= "nav-button1"><%= link_to 'HOME', "/"%></a></li>
+          <li style="float:right"><a class= "nav-button2"><%= link_to 'LOG IN', "/login" %></a></li> 
+          <!-- <li style="float:right"><a class= "nav-button3"> link_to 'LOG OUT', "/login" %></a></li> #if current_user? -->
```
Comment (madhalle) [inline app/views/layouts/application.html.erb:51]: this is where current user method would effect nav bar.

File: spec/welcome/home_page_spec.rb [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+RSpec.describe "Welcome Page" do
+  it "should have landing page, log in link and todays neos" do
+    visit "/"
+
+    expect(page).to have_content("WELCOME TO HORROR SCOPES")
+    expect(page).to have_content("TODAYS TRAGEDY AS PREDICTED BY THE PLANETS")
+    expect(page).to have_content("LOG IN")
+    expect(page).to have_content("HOME")
+    expect(page).to have_css(".cathulu")
```
Comment (madhalle) [inline spec/welcome/home_page_spec.rb:11]: can't figure out how to get the test to actually make sure the image is there. It should up on localhost but needs to be tested.

File: app/assets/stylesheets/application.css [hunk 7]
```diff
@@ -13,3 +13,70 @@
  *= require_tree .
  *= require_self
  */
+.welcome-body {
+   background-color: #101010;
+ }
+ .welcome-page-header-content {
+   padding-top: 200px;
+ }
```
Comment (elguapogordo) [inline app/assets/stylesheets/application.css:21]: the other pages are going to be borrowing a lot of css from here, too :-)

File: app/views/welcome/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+          <p>TODAYS TRAGEDY AS PREDICTED BY THE PLANETS</p>
+        </div>
+    </div>
+  </div>
+  <div class="floaty-cathulu-container">
+    <div class="cathulu">
+      <%= image_tag("icons/whiter_cathulu.png", width: 400) %>
+    </div>
+  </div>
+</body>
```
Comment (elguapogordo) [inline app/views/welcome/index.html.erb:16]: very, very happy with how this turned out!

**Pull request merged**